From 8b354217b9d72f8376c420b30dbd61a5d0cfa5cf Mon Sep 17 00:00:00 2001
From: Pierre Wielders <developper@wielders.net>
Date: Tue, 4 Apr 2023 16:43:19 +0200
Subject: [PATCH] Touch evnt is not working if you expect it to be a  scroll.

Issue raised, waiting for a proper fix. For now, we do not need the scroll if we are using touch!
---
 Source/WebKit/UIProcess/API/wpe/WPEView.cpp | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Source/WebKit/UIProcess/API/wpe/WPEView.cpp b/Source/WebKit/UIProcess/API/wpe/WPEView.cpp
index 67e39c4b68ad3..6925f2a6597e6 100644
--- a/Source/WebKit/UIProcess/API/wpe/WPEView.cpp
+++ b/Source/WebKit/UIProcess/API/wpe/WPEView.cpp
@@ -171,16 +171,6 @@ View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseC
             auto& page = view.page();
 
             WebKit::NativeWebTouchEvent touchEvent(event, page.deviceScaleFactor());
-
-            auto& scrollGestureController = *view.m_scrollGestureController;
-            if (scrollGestureController.isHandling()) {
-                const struct wpe_input_touch_event_raw* touchPoint = touchEvent.nativeFallbackTouchPoint();
-                if (touchPoint->type != wpe_input_touch_event_type_null && scrollGestureController.handleEvent(touchPoint)) {
-                    page.handleWheelEvent(WebKit::NativeWebWheelEvent(scrollGestureController.axisEvent(), page.deviceScaleFactor(), scrollGestureController.phase(), WebWheelEvent::Phase::PhaseNone));
-                    return;
-                }
-            }
-
             page.handleTouchEvent(touchEvent);
         },
 #endif
